src/main/java/duke/EmptyLineException.java

```diff
@@ -0,0 +1,9 @@
+package duke;
+
+public class EmptyLineException {
```

Comment:
since this is a custom exception, you need to write Javadoc comments to indicate its purpose in the codebase